content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -0,0 +1,429 @@
+===========
+Help Center
+===========
+
+Odoo *Helpdesk* integrates with the *Forums*, *eLearning*, and *Knowledge* apps to create the *Help
+Center*.
```

Comment:
Since "help center" is not an app/module, an element on the UI, or a proper noun and is just an umbrella term for the combined resources and features between Helpdesk, Knowledge, Forums, and eLearning I believe it should be sentence case throughout the document :)
```suggestion
===========
Help center
===========

Odoo *Helpdesk* integrates with the *Forums*, *eLearning*, and *Knowledge* apps to create the *help
center*.
```